### DISCUSSION TITLE
Improve website

### DISCUSSION THREAD

**Discussion by tiagoagenor** [nhibernate/nhibernate-core#2640] [General]
Today was the first time that I’m mixing nhibernate. And I found the site with a lot of information. I think I wanted the site better interesting. If you need help I can help.

**Comment by oskarb** [comment]
Hi,

The web site source is also handled in github (https://github.com/nhibernate/nhibernate.github.io) so if you have specific suggestions that you want to discuss, or if you want to open pull requests with changes, you are free to do so. Of course, if you are thinking of big changes, it's could be a big effort to implement and evaluate.
